public/js/main.js

```diff
@@ -30,7 +30,9 @@ connectClient(response => {
   // otherwise, just show the toolbox.
   if (hasSelectedTab()) {
     const selectedTab = getSelectedTab(store.getState().tabs.get("tabs"));
-    debugTab(selectedTab.toJS(), actions).then(renderToolbox);
+    debugTab(selectedTab.toJS(), actions).then(() => {
+      setTimeout(renderToolbox, 0);
```

Comment:
I'll remove it, I was only using it will heavily debugging. Calling `renderToolbox` within `then` makes the entire rendering happen within a promise, so any errors will be gobbled up. We show unhandled errors in devtools but only when the promise is garbage collected, so you have to wait a few seconds (or even longer in worst cases).

Anyway, I hate promises because they are like a virus. They spread everywhere.